config/boot.rb

```diff
@@ -1,3 +1,4 @@
 ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
 
 require 'bundler/setup' # Set up gems listed in the Gemfile.
+# require 'bootsnap/setup' # Speed up boot time by caching expensive operations.
```

Comment:
This is the only change I made aside from what was automatically done in `app:update`. Since boot time is very fast on this app, I didn't want to add the bootsnap gem, as outlined in the docs: https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#upgrading-from-rails-5-1-to-rails-5-2-bootsnap